pkg/binding/bindingpolicy-resolution.go

```diff
@@ -113,6 +121,19 @@ func (resolution *bindingPolicyResolution) setDestinations(destinations sets.Set
 	resolution.destinations = destinations
 }
 
+// getObjectKeys returns the Keys of the objects in the resolution.
+func (resolution *bindingPolicyResolution) getObjectKeys() []*util.Key {
```

Comment:
Note to self: why not `Set[util.Key]` ?

Reply:
That indeed should happen, along with the keys rework.

For the moment this is very convenient (and aligned with the resolver).

Reply:
More importantly, the ultimate consumer does not really care, it is a loop in `func (c *Controller) requeueSelectedWorkloadObjects`.